crates/pathfinder/src/sierra.rs

```diff
@@ -22,7 +22,7 @@ pub fn compile_to_casm(sierra_definition: &[u8]) -> anyhow::Result<Vec<u8>> {
     validate_compatible_sierra_version(&sierra_class, ListSelector::DefaultList)?;
 
     let casm_class =
-        CasmContractClass::from_contract_class(sierra_class).context("Compiling to CASM")?;
+        CasmContractClass::from_contract_class(sierra_class, true).context("Compiling to CASM")?;
```

Comment:
I've just realized this is StarkWare code? Is that.. safe to do?

Reply:
This is the actual compilation step. What do you mean by "safe"?

Reply:
Ah right, that makes sense. Hmm. Am I right in saying its a very heavy dep? I wonder if they can trim it down to only the compilation.

But okay, that's not our problem right now. Sorry, I think it just never occurred to me that we of course need some starknet rust code.